lib/cabinet.js

```diff
@@ -10,14 +10,14 @@ Cabinet.SIGNATURE = 0x4643534D
 Cabinet.MSZIP_SIGNATURE = 0x4B43
 
 Cabinet.COMPRESSION = {
-  NONE: 0x0000,
-  MSZIP: 0x0100,
-  QUANTUM: 0x0200,
-  LZX: 0x0300,
+  NONE: 0,
+  MSZIP: 1,
+  QUANTUM: 2,
+  LZX: 3
   // NOTE: High 8 bits store compression extra
   // (for LZX the value between 15-21 [0x0F-0x15])
-  // LZX21: 0x0315,
-  // LZX15: 0x030F,
+  // LZX21: 0x1503,
+  // LZX15: 0x150F,
```

Comment:
As `0x03` is the LZX compression type and `0x0F` and `0x15` the high "extra" fields for compression parameters, these should be as below I believe
```suggestion
  // LZX21: 0x1503,
  // LZX15: 0x0F03,
```